<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMLSelectLayersDialog</class>
 <widget class="QDialog" name="GMLSelectLayersDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wizualizacja GML - wybór warstw</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTreeWidget" name="layers_tree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>760</width>
     <height>570</height>
    </rect>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>300</number>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Nazwa warstwy</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Typ geometrii</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Liczba obiektów</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="select_all_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>615</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zaznacz/Odznacz wszystkie</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="load_layers_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>620</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wczytaj</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
